CMakeLists.txt

```
@@ -1028,6 +1031,11 @@ if(APPLE)
 		)
 endif()
 
+if(USE_DISCORD_PRESENCE)
+  message(STATUS "Using static DiscordRPC from Externals")
+  add_subdirectory(Externals/discord-rpc)
```

```suggestion
  add_subdirectory(Externals/discord-rpc EXCLUDE_FROM_ALL)
```
https://github.com/dolphin-emu/dolphin/pull/9305